docs/operators/jdbcPOJOInsertOutputOperator.md

```diff
@@ -0,0 +1,179 @@
+| *databaseDriver* |JDBC Driver class for connection to JDBC Store. This driver should be present in the class path | String | Yes | N/A |
+| *databaseUrl* |["Database URL"](http://www.roseindia.net/tutorial/java/jdbc/databaseurl.html) of the form jdbc:subprotocol:subname | String | Yes | N/A |
+| *userName* | Name of the user configured in the database | String | Yes | N/A |
+| *password* | Password of the user configured in the database | String | Yes | N/A |
+
+Those properties can be set like this:
+
+```xml
+<property>
+  <name>dt.operator.{OperatorName}.prop.batchSize</name>
+  <value>500</value>
+</property>
+
+<property>
+  <name>dt.operator.{OperatorName}.prop.store.databaseDriver</name>
+  <value>com.mysql.jdbc.Driver</value>
+</property>
+
+<property>
+  <name>dt.operator.{OperatorName}.prop.store.databaseUrl</name>
+  <value>jdbc:mysql://localhost:3306/mydb</value>
+</property>
+
+<property>
+  <name>dt.operator.{OperatorName}.prop.store.userName</name>
+  <value>myuser</value>
+</property>
+
+<property>
+  <name>dt.operator.{OperatorName}.prop.store.password</name>
+  <value>mypassword</value>
+</property>
+```
+### Abstract Methods
+These methods are defined as abstract in AbstractJdbcTransactionableOutputOperator `void setStatementParameters(PreparedStatement statement, T tuple)`:Sets the parameter of the insert/update statement with values from the tuple.
+`String getUpdateCommand()`:Gets the statement which insert/update the table in the database.
+
+
+## AbstractJdbcPOJOOutputOperator
+This is the abstract implementation extending the functionality of AbstractJdbcTransactionableOutputOperator that serves as base class for inserting rows in a table using a JDBC store. It has the definition for the abstract methods in AbstractJdbcTransactionableOutputOperator. It can be further extended to modify functionality or add new capabilities. This class has an input port, to recieve the records in the form of tuples, so concrete sub classes won't need to provide the same, and processes/inserts each input tuple as a database table record one by one . You need to set the input port attribute TUPLE_CLASS to define your [POJO](https://en.wikipedia.org/wiki/Plain_Old_Java_Object) class name to define Object type.
+
+### <a name="AbstractJdbcPOJOOutputOperatorProps"></a>Properties of AbstractJdbcPOJOOutputOperator
+Several properties are available to configure the behavior of this operator and they are summarized in the table below.
+
+| **Property** | **Description** | **Type** | **Mandatory** | **Default Value** |
+| -------- | ----------- | ---- | ------------------ | ------------- |
+| *tablename* | Name of the table where data is to be inserted | String | Yes | N/A |
+| *fieldInfos*| JdbcFieldInfo maps a store column to a POJO field name| List | Yes | N/A |
+
+Those properties can be set like this:
+
+```xml
+<property>
+  <name>dt.operator.{OperatorName}.prop.tablename</name>
+  <value>tablename</value>
```

Comment:
Please use something that resembles an actual table name such as "ResultTable"; otherwise, users may be confused by the fact that this value is identical to the last element  of the <name> element above.